pom.xml

```diff
@@ -263,7 +263,7 @@
                             <goal>generate</goal>
                         </goals>
                         <configuration>
-                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-payment-instruments-service/main/api-spec/api.yaml</inputSpec>
+                            <inputSpec>https://raw.githubusercontent.com/pagopa/pagopa-ecommerce-payment-instruments-service/CHK-905-afm/api-spec/api.yaml</inputSpec>
```

Comment:
👀

Reply:
Can we restore the endpoint to the main branch?